marketplaceItems/audioFocus/README.md

```diff
@@ -0,0 +1,48 @@
+# AUDIO FOCUS
+
+## Description
+
+This is a simple app that allows you to choose an avatar and solo them.
+You can either select in Desktop by clicking, or in HMD by triggering with either controller
+
+## Releases
```

Comment:
Please make a new Release to S3 and add details of that release here. That release will be the one associated with this merged PR, and will be the one submitted to Marketplace.